pymc/model_graph.py

```diff
@@ -460,6 +464,9 @@ def make_graph(
     node_formatters = update_node_formatters(node_formatters)
 
     graph = graphviz.Digraph(name)
+    if graph_attrs is not None:
+        graph.attr(**graph_attrs)
+
```

Comment:
```suggestion
    graph = graphviz.Digraph(name, graph_attr=graph_attrs)
```

Maybe a bit more direct? From a quick look at the docs, it seems that `graph_attr` can take None, so there's no checking necessary.